types/office-js-preview/index.d.ts

```diff
@@ -58941,6 +59374,70 @@ declare namespace Excel {
         interface SlicerItemCollectionData {
             items?: Excel.Interfaces.SlicerItemData[];
         }
+        /** An interface describing the data returned by calling `linkedDataType.toJSON()`. */
```

Comment:
Again, why describe the interface in terms of its role in calling code? Let's describe what it is instead.

Reply:
This seems to be a consistent issue with all these `*Data` interfaces.

FWIW, we don't include these `*Data` classes in the TOC and the only way to see them is through a link from `*.toJSON`. Changing them will be something we'll need to work with the office-js-api team to fix. Is it worth pursuing?

Reply:
> This seems to be a consistent issue with all these `*Data` interfaces.
> 
> FWIW, we don't include these `*Data` classes in the TOC and the only way to see them is through a link from `*.toJSON`. Changing them will be something we'll need to work with the office-js-api team to fix. Is it worth pursuing?

No. Not worth pursuing.